### PR TITLE
Creating hello microverse and adding readme with the corrections in the professional presentation of the repository

### PR DESCRIPTION
**Pull request by AndresEspi**
### **Achievements**

- Add the liters for HTML and CSS
- I gave the repository an MIT license 
- Add a test.md file which is omitted by the .gitignore file
- Add an index.html file that displays a "hello microverse" message and another style.css that changes the font color of the message 
- Fixed syntax errors in the README.md file
- Previous pull request: https://github.com/AndresEspi/hello-world/pull/1  I hope this will help you

### CODE REVIEW

File: README.md
Comment (Alusp) [inline README.md:1]: - [ ] [Required] Good job in implementing the changes, however, As requested by the previous reviewers, Please do not remove the major sections in the readme file, the only optional section is the FAQ Section as indicated in the Project requirement. Please use [this](https://github.com/microverseinc/readme-template) microverse template and only update the sections according to your Project. :slightly_smiling_face: 
- [ ] [Required]Please as mentioned by the previous reviewers, kindly ensure to remove the commented words in the Readme template as they are just meant for you to know how to update it. :+1:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @AndresEspi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

 ## Highlights

- [ ] Good PR Summary and title :white_check_mark: 
- [ ] No linter Errors :white_check_mark: 
- [ ] Good GitHub Flow  :white_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please do not delete an unapproved Pull request, even if unapproved, it can contain valuable insights, suggestions, and feedback that contribute to the learning process and improvement of code quality. By deleting unapproved pull requests, you may miss out on valuable learning opportunities and overlook potential enhancements to the codebase.
- Please mention that pull request link in this pull request summary in the future, so the next reviewer will be able to click on the link and be redirected to that pull request to check the previously required changes. To add a link in a PR Summary you will just write `Previous Pull Request: #1` I hope this will help you.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rica213** [comment]
# 🟢 STATUS: APPROVED
Hi @AndresEspi, 
My name is Larissa. I am your code reviewer 🥰

Thank you for implementing all the required changes by the previous reviewers ❤️❤️❤️
Your hard work paid off because your project is complete! 👯  Now let's merge it 🛸 🛸 
Congratulations! 🎉 



## Highlights
- [ ] Every requirement is implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
Hey @AndresEspi, 

According to this review [summary](https://github.com/AndresEspi/hello-world/pull/2#issuecomment-1572969990), Your project has been approved: :congratulations:

**Pull request merged**